test/integration/workarea/storefront/gift_card_integration_test.rb

```diff
@@ -5,6 +5,10 @@ module Storefront
     class GiftCardIntegrationTest < Workarea::IntegrationTest
       setup :tax, :product, :shipping_service, :gift_card
 
+      def next_year
```

Comment:
I had to add this here to fix the tests after fixing the initial build...somehow the `v3.3-stable` merge didn't work correctly and there's no `next_year` method for this class to use. I can make a ticket/update to the legacy branch but I wanted to just demonstrate that the build would pass if this method was here.

Reply:
I thought we added `next_year` to the older versions of base that this would be compatible with?

Reply:
something weird seems to have happened in the merge on v3.3-stable, because i don't see the method. but it could have been just a mistake on my part.